erpnext/hr/doctype/leave_application/leave_application.js

```diff
@@ -85,6 +86,9 @@ frappe.ui.form.on("Leave Application", {
 				frm.set_value('employee', perm['Employee'].map(perm_doc => perm_doc.doc)[0]);
 			}
 		}
+		if((!frm.doc.__islocal) && frm.doc.employee) {
+			 $('[data-fieldname="employee_name"]')[0].hidden = true
+		}
```

Comment:
```suggestion
```
- Any reason for hiding this ?

Reply:
There is a field named
1. employee 
2. employee_name 
At the time of New Record employee code is shown in employee field and name in employee_name
but at the time of Edit or view in both the fields employee name is showing and it is duplicate so i was hidding the employee_name

Reply:
- This is a hack.
- Do not delete the field. just hide it in doctype